### ISSUE TITLE
Support for the "muted" attribute (Vimeo)

### ISSUE THREAD

**Issue by alex-mikhaylov** [asciidoctor/asciidoctor#3014]
Hello,

First of all, thanks a lot for an awesome tool!

I'm playing around with Vimeo videos and can't make the "autoloop" attribute work. The Vimeo [documentation](https://help.vimeo.com/hc/en-us/articles/115004485728-Autoplaying-and-looping-embedded-videos) says that we need to add the "muted" attribute to the video as well. 

If I add the muted attribute (e.g. video::12345678[vimeo, width=850, height=450, options="muted,autoplay,loop"]), the autoplay doesn't work. Am I missing anything?

**Comment by mojavelinux** [comment]
The Asciidoctor HTML converter does not currently recognize the `mute` option. I'll add support for it.

Please note that the limitation you describe only affects Chrome. The video does not have be muted in order for autoplay to work in Firefox.

**Comment by mojavelinux** [comment]
I've decided to change the option name to `muted` (instead of `mute`) since that's the name HTML 5 has standardized (see https://dev.w3.org/html5/spec-author-view/video.html#attr-media-muted). It's not consistent with the other option names since it's an adjective instead of a verb. However, I think deviating from HTML would make it harder to remember. Plus, the Vimeo param is also named `muted`. (On the other hand, the YouTube param is named `mute`).

I've also added support for the `muted` option to YouTube videos as well.
